src/ctim/schemas/investigation.cljc

```diff
@@ -14,7 +14,12 @@
   c/sourced-object-entries
   c/describable-entity-entries
   (f/required-entries
-   (f/entry :type InvestigationIdentifier)))
+   (f/entry :type InvestigationIdentifier))
+  (f/optional-entries
+   (f/entry :object_ids (f/seq-of (f/str)))
+   ;; "type:value"
+   (f/entry :investigated_observables (f/seq-of (f/str)))
+   (f/entry :targets (f/seq-of c/IdentitySpecification))))
```

Comment:
Could you add a description to these new fields?

Ex:
```clojure
(f/entry :targets (f/seq-of c/IdentitySpecification)
         :description "Investigated target devices")
```

Reply:
Yes. I'm still working on this but I'll apply this suggestion.